docs/source/en/training/dreambooth.mdx

```diff
@@ -549,17 +630,50 @@ accelerate launch train_dreambooth.py \
   --instance_data_dir=$INSTANCE_DIR \
   --output_dir=$OUTPUT_DIR \
   --instance_prompt="a photo of sks dog" \
-  --resolution=64 \ # The input resolution of the IF unet is 64x64
+  --resolution=64 \
   --train_batch_size=4 \
   --gradient_accumulation_steps=1 \
   --learning_rate=1e-7 \
   --max_train_steps=150 \
   --validation_prompt "a photo of sks dog" \
   --validation_steps 25 \
-  --text_encoder_use_attention_mask \ # IF expects attention mask for text embeddings
-  --tokenizer_max_length 77 \ # IF expects an override of the max token length
-  --pre_compute_text_embeddings \ # Pre compute text embeddings to that T5 doesn't have to be kept in memory
+  --text_encoder_use_attention_mask \
+  --tokenizer_max_length 77 \
+  --pre_compute_text_embeddings \
   --use_8bit_adam \ # 
   --set_grads_to_none \
-  --skip_save_text_encoder # do not save the full T5 text encoder with the model
-```
\ No newline at end of file
+  --skip_save_text_encoder
+```
+
+### IF Stage II Full Dreambooth
+
+`--learning_rate=1e-8`: Even lower learning rate.
+
+`--resolution=256`: The upscaler expects higher resolution inputs
+
+```sh
+export MODEL_NAME="DeepFloyd/IF-II-L-v1.0"
+export INSTANCE_DIR="dog"
+export OUTPUT_DIR="dreambooth_dog_upscale"
+export VALIDATION_IMAGES="image_1.png image_2.png image_3.png image_4.png"
+
+accelerate launch train_dreambooth.py \
+  --report_to wandb \
+  --pretrained_model_name_or_path=$MODEL_NAME \
+  --instance_data_dir=$INSTANCE_DIR \
+  --output_dir=$OUTPUT_DIR \
+  --instance_prompt="a sks dog" \
+  --resolution=256 \
+  --train_batch_size=2 \
+  --gradient_accumulation_steps=2 \
+  --learning_rate=1e-8 \
+  --max_train_steps=2000 \
+  --validation_prompt="a sks dog" \
+  --validation_steps=150 \
+  --checkpointing_steps=500 \
+  --pre_compute_text_embeddings \
+  --tokenizer_max_length=77 \
+  --text_encoder_use_attention_mask \
+  --validation_images $VALIDATION_IMAGES \
+  --class_labels_conditioning timesteps
```

Comment:
The default should always be to push to the Hub. Let's add this here and make sure it works:
```suggestion
  --class_labels_conditioning timesteps \
  --push_to_hub
```